lib/ansible/cli/galaxy.py

```diff
@@ -174,7 +174,7 @@ def execute_init(self):
         init_path  = self.get_opt('init_path', './')
         force      = self.get_opt('force', False)
         offline    = self.get_opt('offline', False)
-
+        
```

Comment:
Sorry. Still figuring this out. The lines were unintentional, I didn't know how to get rid of them.